gobblin-data-management/src/main/java/gobblin/data/management/copy/replication/ReplicationUtils.java

```diff
@@ -0,0 +1,112 @@
+package gobblin.data.management.copy.replication;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.common.base.Optional;
+import com.google.common.base.Preconditions;
+import com.typesafe.config.Config;
+
+import gobblin.data.management.copy.CopyableDataset;
+import gobblin.dataset.DatasetsFinder;
+
+
+/**
+ * Utility class to build different java class from {@link com.typesafe.config.Config} 
+ * @author mitu
+ *
+ */
+public class ReplicationUtils {
+  public static final String METADATA = "metadata";
+  public static final String METADATA_JIRA = "jira";
+  public static final String METADATA_OWNER = "owner";
+  public static final String METADATA_NAME = "name";
+
+  public static final String REPLICATION_SOURCE = "source";
+  public static final String REPLICATION_REPLICAS = "replicas";
+  public static final String REPLICATOIN_REPLICAS_LIST = "list";
+
+  public static final String REPLICATION_LOCATION_TYPE_KEY = "type";
+
+  public static final String DATAFLOWTOPOLOGY = "dataFlowTopology";
+
+  public static ReplicationMetaData buildMetaData(Config config) {
```

Comment:
Can you move these method to their respective classes? e.g. `ReplicationMetadata.fromConfig(Config)`.